.github/workflows/push_pr.yml

```diff
@@ -28,11 +28,9 @@ jobs:
   compile_and_test:
     strategy:
       matrix:
-        flink: [ 1.18-SNAPSHOT ]
-        jdk: [ '8, 11, 17' ]
+        flink: [ 1.19-SNAPSHOT ]
+        jdk: [ '8, 11, 17, 21' ]
         include:
-          - flink: 1.19-SNAPSHOT
-            jdk: '8, 11, 17, 21'
           - flink: 1.20-SNAPSHOT
             jdk: '8, 11, 17, 21'
```

Comment:
I think similar story 
since now java is same for all it could be replaced with
```
flink: [ 1.19-SNAPSHOT, 1.20-SNAPSHOT ]
jdk: [ '8, 11, 17, 21' ]
```

and we could completely remove everything under `include:` and `include:` itself

Reply:
@snuyanzin Since we are here, is there a reason we are running PR CI tests against snapshot versions, should we replace them with `1.19.1` and `1.20.0` ?

Reply:
IIRC it allows to catch breaking changes faster before they released instead of waiting for the release and then receive a bunch of failures

I remember it allowed to catch already several things like that

Reply:
I see, thanks for the info